oraclizeAPI_0.5.sol

```diff
@@ -226,59 +543,100 @@ library CBOR {
         }
     }
 
-    function encodeIndefiniteLengthType(Buffer.buffer memory _buf, uint8 _major) private pure {
+    function encodeIndefiniteLengthType(
+        Buffer.buffer memory _buf,
+        uint8 _major
+    )
+        private
+        pure
+    {
         _buf.append(uint8((_major << 5) | 31));
     }
 
-    function encodeUInt(Buffer.buffer memory _buf, uint _value) internal pure {
+    function encodeUInt(
+        Buffer.buffer memory _buf,
+        uint256 _value
+    )
+        internal
+        pure
+    {
         encodeType(_buf, MAJOR_TYPE_INT, _value);
     }
 
-    function encodeInt(Buffer.buffer memory _buf, int _value) internal pure {
+    function encodeInt(
+        Buffer.buffer memory _buf,
+        int _value
+    )
+        internal
+        pure
+    {
         if (_value >= 0) {
             encodeType(_buf, MAJOR_TYPE_INT, uint(_value));

```

Comment:
Is it possible to cast to uint256?

Reply:
sure, but it's an external lib, and for easier import let's leave it as is.